src/mono/netcore/sample/Android/Program.csproj

```diff
@@ -0,0 +1,67 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <OutputPath>bin</OutputPath>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <TargetArchitecture Condition="'$(TargetArchitecture)'==''">x64</TargetArchitecture>
+    <RuntimePackDir>$(ArtifactsDir)bin\lib-runtime-packs\runtimes\android-$(TargetArchitecture)</RuntimePackDir>
+    <MonoRuntimeDir>$(ArtifactsDir)bin\mono\Android.$(TargetArchitecture).$(Configuration)</MonoRuntimeDir>
```

Comment:
why do we need this? the runtime pack includes the mono runtime.

Reply:
`build.sh -subset Mono` doesn't update mono pieces in the runtime pack so this line is useful for development purposes

Reply:
That's not really sustainable once we switch to the publish pipeline though. People will need to get familiar with running `build.sh -subset mono+libs.pretest` or we need to make this automatic.